lib/camunda-cloud/features/properties-provider/parts/HeadersProps.js

```diff
@@ -38,3 +38,13 @@ export default function(group, element, bpmnFactory, translate) {
   }
 
 }
+
+
+// helpers ////////////////
+
+function canHaveHeaders(element) {
+  return isAny(element, [
+    'bpmn:ServiceTask',
+    'bpmn:UserTask'
+  ]);
+}
```

Comment:
I believe this is a clear sign that we need a `bpmn-js` extension in `zeebe-bpmn-moddle`. With the current state, I can morph a User Task to a Receive Task and `zeebe:taskHeaders` stay in the extension elements.

Reply:
Good catch, but this still exists outside of this PR, right? E.g. By morphing a Service Task to a Receive Task.

Shall we create an issue on `zeebe-bpmn-moddle`?

Reply:
Seems like we already [have this extension](https://github.com/zeebe-io/zeebe-bpmn-moddle/blob/master/lib/extension.js#L62), but no [`allowedIn` meta for headers](https://github.com/zeebe-io/zeebe-bpmn-moddle/blob/master/resources/zeebe.json#L111).

Reply:
Oh, I missed that 🤪  Let's have a separate issue then. I'll create it.

Reply:
[Done](https://github.com/zeebe-io/zeebe-bpmn-moddle/issues/7)